src/controllers/v1/entities.js

```diff
@@ -57,28 +59,403 @@ module.exports = class Entities extends Abstract {
   ]
 */
 
-  /**
-   * List all the entities.
-   * @method
-   * @name list 
-   * @returns {JSON} - List of all entities.
-   */
-
-  async list(req) {
-    return new Promise(async (resolve, reject) => {
-      try {
-
-        let result = await entitiesHelper.list("all", { name: 1 })
-
-        return resolve(result)
-
-      } catch (error) {
-        return reject({
-          status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
-          message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
-          errorObject: error
-        })
-      }
-    })
-  }
-}
\ No newline at end of file
+	/**
+	 * Find all the entities.
+	 * @method
+	 * @name find
+	 * @returns {JSON} - List of all entities.
+	 */
+
+	find(req) {
+		return new Promise(async (resolve, reject) => {
+			try {
+				let entityData = await entitiesQueries.entityDocuments(req.body.query, req.body.projection)
+				return resolve({
+					success: true,
+					message: CONSTANTS.apiResponses.ASSETS_FETCHED_SUCCESSFULLY,
+					result: entityData,
+				})
+			} catch (error) {
+				return reject({
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+					errorObject: error,
+				})
+			}
+		})
+	}
+
+	relatedEntities(req) {
```

Comment:
@MallanagoudaB api description is missing. add descriptions to all if it is missing..